### ISSUE TITLE
json_encode doesn't escape quotes

### ISSUE THREAD

**Issue by skorowarkin** [matt-42/iod#17]
json_string s{"\""};
json_encode( s ); // got unescaped "

Actually noted in Silicon: json_encode'ing some D object with string containing quotes returns invalid JSON.

**Comment by matt-42** [comment]
Yes, this is missing. I finished a full unicode encode/decoder last month.
I'll integrate it into silicon.

https://github.com/iodcpp/metajson/blob/master/metajson/unicode.hh

**Comment by skorowarkin** [comment]
Okay, that's nice

**Comment by matt-42** [comment]
Fixed with c5d41008a95b0e5ad0a9ea60f352dc230ed81ada
